semaphoreci.sh

```diff
@@ -36,7 +36,14 @@ source ci.sh
 # Always source a DMD instance
 ################################################################################
 
-install_d "$DMD"
+# FIXME: v2.082.0 has a broken DUB which fails the CI
+# Remove this when a fixed v2.082.1 is released
+# See https://github.com/dlang/dub/issues/1551
+if [ "$DMD" == "dmd" ]; then
+    install_d "dmd-2.081.2"
```

Comment:
We don’t want to install Dub separately and still use DMD 2.082.0?

Reply:
There's already all the logic in place to get a specific version in `install.sh`, but nothing to get `dub`. It would be a much bigger investment (and waste, IMHO) of time to do the later, and not of much help.

I already submitted a PR to dub to fix the bug, all I'm waiting for is for someone with the right credentials to pull it and make a release. This just unblocks DMD development.